package.json

```diff
@@ -55,7 +55,6 @@
     "@types/luxon": "^1.24.0",
     "@typescript-eslint/eslint-plugin": "^3.0.1",
     "@typescript-eslint/parser": "^3.0.1",
-    "eslint": "^7.1.0",
```

Comment:
Does it work well without the version 7 of eslint? I had an issue with typescript and had to install these and include the `SKIP_PREFLIGHT_CHECK=true`  in .env but it that works for you I'll try again because it makes it easier

Reply:
I don't know why you needed to upgrade to eslint 7, but if the fix is changing .env, are we sure it's worth it? Feels like a hack and would then need to be set in every local instance of the project (for instance I had no idea), plus in cloud builds.